actions/packages.py

```diff
@@ -233,11 +234,16 @@ def __transact_args__(self, transaction_data):
                 assert False, "Unknown package transaction action."
         return args
 
-    def patch_install(self, patch_list):
+    def patch_install(self, patch_list, updatestack_only=False,  allow_vendor_change=False):
```

Comment:
`updatestack_only` looks like a copy&paste error. It has nothing to do with the feature